cub/device/dispatch/dispatch_select_if.cuh

```diff
@@ -186,7 +487,21 @@ struct device_select_policy_hub
                                                   detail::fixed_delay_constructor_t<350, 450>>;
     };
 
-    using MaxPolicy = Policy350;
+    struct Policy900 : ChainedPolicy<900, Policy900, Policy350>
+    {
+      static constexpr bool flagged = std::is_same<FlagT, NullType>::value == false;
```

Comment:
That condition is a bit surprising to read. 
```suggestion
      static constexpr bool flagged = !std::is_same<FlagT, NullType>::value;
```

Reply:
My eyes don't see `!` among 73 other characters for some reason :)